index.js

```diff
@@ -29,8 +29,8 @@ function translate(text, opts) {
     opts.to = languages.getCode(opts.to);
 
     return token.get(text).then(function (token) {
-        var tld = 'com';
-        var url = 'https://translate.google.' + {tld} + '/translate_a/single';
+        opts.tld = opts.tls || 'com';
```

Comment:
it's better to move this up and pass opts to `token.get(text)` as we need tld for getting token also.

Reply:
I've done the changes you suggested on the API side of things.

Reply:
This has been resolved as of 2.8.0, so I'll go ahead and resolve this.